pkg/services/publicdashboards/service/service_wapper.go

```diff
@@ -43,3 +42,16 @@ func (pd *PublicDashboardServiceWrapperImpl) FindByDashboardUid(ctx context.Cont
 
 	return pubdash, nil
 }
+
+func (pd *PublicDashboardServiceWrapperImpl) DeleteByPublicDashboard(ctx context.Context, pubdash *PublicDashboard) error {
+	affectedRows, err := pd.store.Delete(ctx, pubdash.OrgId, pubdash.Uid)
```

Comment:
The `orgId` shouldn't be necessary to delete a pubdash, only the uid. This code is old I am making this comment just to be aware of it 😅